stubs/redis/redis/commands/core.pyi

```diff
@@ -28,27 +33,132 @@ class CoreCommands(Generic[_StrType]):
         reset: bool = ...,
         reset_keys: bool = ...,
         reset_passwords: bool = ...,
+        **kwargs,
     ) -> bool: ...
-    def acl_users(self) -> list[str]: ...
-    def acl_whoami(self) -> str: ...
+    def acl_users(self, **kwargs) -> list[str]: ...
+    def acl_whoami(self, **kwargs) -> str: ...
+
+class ManagementCommands:
+    def bgrewriteaof(self, **kwargs): ...
+    def bgsave(self, schedule: bool = ..., **kwargs): ...
+    def role(self): ...
+    def client_kill(self, address: str, **kwargs) -> bool: ...
+    def client_kill_filter(
+        self,
+        _id: Any | None = ...,
+        _type: Any | None = ...,
+        addr: Any | None = ...,
+        skipme: Any | None = ...,
+        laddr: Any | None = ...,
+        user: Any | None = ...,
+        **kwargs,
+    ): ...
+    def client_info(self, **kwargs): ...
+    def client_list(self, _type: str | None = ..., client_id: list[str] = ..., **kwargs) -> list[dict[str, str]]: ...
+    def client_getname(self, **kwargs) -> str | None: ...
+    def client_getredir(self, **kwargs): ...
+    def client_reply(self, reply, **kwargs): ...
+    def client_id(self, **kwargs) -> int: ...
+    def client_tracking_on(
+        self, clientid: Any | None = ..., prefix=..., bcast: bool = ..., optin: bool = ..., optout: bool = ..., noloop: bool = ...
+    ): ...
+    def client_tracking_off(
+        self, clientid: Any | None = ..., prefix=..., bcast: bool = ..., optin: bool = ..., optout: bool = ..., noloop: bool = ...
+    ): ...
+    def client_tracking(
+        self,
+        on: bool = ...,
+        clientid: Any | None = ...,
+        prefix=...,
+        bcast: bool = ...,
+        optin: bool = ...,
+        optout: bool = ...,
+        noloop: bool = ...,
+        **kwargs,
+    ): ...
+    def client_trackinginfo(self, **kwargs): ...
+    def client_setname(self, name: str, **kwargs) -> bool: ...
+    def client_unblock(self, client_id, error: bool = ..., **kwargs): ...
+    def client_pause(self, timeout, all: bool = ..., **kwargs): ...
+    def client_unpause(self, **kwargs): ...
+    def command(self, **kwargs): ...
+    def command_info(self, **kwargs): ...
+    def command_count(self, **kwargs): ...
+    def config_get(self, pattern: str = ..., **kwargs): ...
+    def config_set(self, name, value, **kwargs): ...
+    def config_resetstat(self, **kwargs): ...
+    def config_rewrite(self, **kwargs): ...
+    def dbsize(self, **kwargs) -> int: ...
+    def debug_object(self, key, **kwargs): ...
+    def debug_segfault(self, **kwargs): ...
+    def echo(self, value: _Value, **kwargs) -> bytes: ...
```

Comment:
Is this one intentionally different from other methods that return `str` or some type containing `str`?

Reply:
Another moved method.